_snippets/graphql/add-members-to-tier.mdx

```diff
@@ -37,7 +37,7 @@ mutation addMembersToTier($input: AddMembersToTierInput!) {
         "companyId": "co_123"
       }
     ],
-    "tierId": "tier_123"
+    "tierIdentifier": { "externalId": "tier_123" }
```

Comment:
`tier_123` could be confused with our tier ids. Can you use `XXX` which is what I think we use in the other externalId tier examples we have in the docs?